src/pycram/yarp_utils/yarp_joint_state_updater.py

```diff
@@ -0,0 +1,324 @@
+import math
+import sys
+from time import sleep
+
+from pycram import World
```

Comment:
Change this import to "from ..datastructures.world import World"

Reply:
done